topics/introduction/tutorials/options-for-using-galaxy/slides.html

```diff
@@ -164,3 +221,8 @@
   - data security
   - computational requirements
   - tools installed
+???
+- In conclusion, choosing which Galaxy instance to use depends on multiple factors like: size of datasets, available storage, backup
+- you also need to consider data security, computational requirements, and the tools installed
+
+---
```

Comment:
I think remove this one because it starts an empty slide at the end, not sure how the video pipeline would deal with that.

```suggestion
```